guides/source/active_record_basics.md

```diff
@@ -71,107 +70,153 @@ Convention over Configuration in Active Record
 
 Creating Active Record Models
 -----------------------------
 
-When generating an application, an abstract `ApplicationRecord` class will be
-created in `app/models/application_record.rb`. This is the base class for all
-models in an app, and it's what turns a regular ruby class into an Active Record
-model.
+When generating a Rails application, an abstract `ApplicationRecord` class will
+be created in `app/models/application_record.rb`. The `ApplicationRecord` class
+inherits from [`ActiveRecord::Base`](https://api.rubyonrails.org/classes/ActiveRecord/Base.html) and it's what turns a regular Ruby class into
+an Active Record model.
 
-To create Active Record models, subclass the `ApplicationRecord` class and you're good to go:
+`ApplicationRecord` is the base class for all Active Record models in your app.
+To create a new model, subclass the `ApplicationRecord` class and you're good to
+go:
 
 ```ruby
-class Product < ApplicationRecord
+class Book < ApplicationRecord
 end
 ```
 
-This will create a `Product` model, mapped to a `products` table at the
-database. By doing this you'll also have the ability to map the columns of each
-row in that table with the attributes of the instances of your model. Suppose
-that the `products` table was created using an SQL (or one of its extensions) statement like:
+This will create a `Book` model, mapped to a `books` table in the
+database, where each column in the table is mapped to attributes of the `Book` class. An instance of `Book` can represent a row in the `books` table. The `books` table can be created using an SQL statement like this:
 
 ```sql
-CREATE TABLE products (
+CREATE TABLE books (
   id int(11) NOT NULL auto_increment,
-  name varchar(255),
+  title varchar(255),
+  author varchar(255),
   PRIMARY KEY  (id)
 );
 ```
 
-The schema above declares a table with two columns: `id` and `name`. Each row of
-this table represents a certain product with these two parameters. Thus, you
-would be able to write code like the following:
+Database tables in Rails are typically created using [Active Record Migrations](#migrations) and not raw SQL. A migration for the `books` table above can be generated like this:
+
+```bash
+$ rails generate migration CreateBooks title:string author:string
+```
+
+and results in this:
 
 ```ruby
-p = Product.new
-p.name = "Some Book"
-puts p.name # "Some Book"
+# Note: the `id` column, as the primary key, is automatically created by convention. As are `created_at` and `updated_at` columns.
+
+# /db/migrate/20240220143807_create_books.rb
+class CreateBooks < ActiveRecord::Migration
+  def change
+    create_table :books do |t|
+      t.string :title
+      t.string :author
+
+      t.timestamps
+    end
+  end
+end
 ```
 
+The SQL as well as the migration above declare a table with three columns: `id`,
+`title`, and `author`. Each row of this table can be represented by an instance
+of the `Book` class with the same three attributes, `id`, `title`, and `author`.
```

Comment:
Maybe this line could be better with a colon, like:
```suggestion
of the `Book` class with the same three attributes: `id`, `title` and `author`.
```

or perhaps delete the attributes since they are quoted in line 203-204

```suggestion
of the `Book` class with the same three attributes.
```